src/main/java/de/hysky/skyblocker/config/categories/DungeonsCategory.java

```diff
@@ -307,6 +307,26 @@ public static ConfigCategory create(SkyblockerConfig defaults, SkyblockerConfig
                                 .build())
                         .build())
 
+                // Waypoints for goldor phase in f7/m7
+                .group(OptionGroup.createBuilder().name(Text.translatable("skyblocker.config.dungeons.goldorWaypoints"))
+                        .collapsed(true)
+                        .option(Option.<Boolean>createBuilder()
+                                .name(Text.translatable("skyblocker.config.dungeons.goldorWaypoints.enableGoldorWaypoints"))
+                                .binding(defaults.dungeons.goldor.enableGoldorWaypoints,
+                                        () -> config.dungeons.goldor.enableGoldorWaypoints,
+                                        newValue -> config.dungeons.goldor.enableGoldorWaypoints = newValue)
+                                .controller(ConfigUtils::createBooleanController)
+                                .build())
+                        .option(Option.<Type>createBuilder()
+                                .name(Text.translatable("skyblocker.config.dungeons.goldorWaypoints.waypointType"))
+                                .description(OptionDescription.of(Text.translatable("skyblocker.config.dungeons.goldorWaypoints.waypointType.@Tooltip")))
```

Comment:
No need for extra work for the translators.
```suggestion
                                .description(OptionDescription.of(Text.translatable("skyblocker.config.uiAndVisuals.waypoints.waypointType.@Tooltip")))
```